### PR TITLE
Remove unnecessary concatenation

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Description

Maybe this is unnecessary

## Checklist

- [x] I’ve added tests to confirm my change works.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-js/print/call-arguments.js [hunk 1]
```diff
@@ -117,12 +117,10 @@ function printCallArguments(path, options, print) {
         iterateCallArgumentsPath(path, (argPath, i) => {
           if (shouldGroupFirst && i === 0) {
             printedExpanded = [
-              [
-                print([], { expandFirstArg: true }),
-                printedArguments.length > 1 ? "," : "",
-                hasEmptyLineFollowingFirstArg ? hardline : line,
-                hasEmptyLineFollowingFirstArg ? hardline : "",
-              ],
```
Comment (thorn0) [inline src/language-js/print/call-arguments.js:125]: Every element of the `printedExpanded` array corresponds to an argument of the call. So, for example, `printedExpanded[0]` is the first argument. `printedExpanded[0]` is used in the code below and your change changes the meaning of that code.

Comment (sosukesuzuki) [reply src/language-js/print/call-arguments.js:125]: Oh..., sorry can you show me an example of how it would actually change the meaning of the code?

Comment (thorn0) [reply src/language-js/print/call-arguments.js:125]: I'll try, but this doesn't look easy

Comment (sosukesuzuki) [reply src/language-js/print/call-arguments.js:125]: Okay, thanks

Comment (thorn0) [reply src/language-js/print/call-arguments.js:125]: **Prettier 2.3.0**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAjAlgMwgJygYR3SgHM4cBZCAZwAs4ATABXIAcAbOADwGUBDAO4NyuAkVLkqdRixwdu-IeQCKA8gAp1ASgAEAXgB8O4AF8ANDoBMABmtaQZkBFYx00aslB8cOCAKbeCB4ofOwCfACeHo6oOHxgANZwMDys8cQkyDA4AK5wjnAAtqiMwgwAMnykOXxkAGK4hXwwrqTIIHw5MBAOILQwhewA6rTo8NRpYHA8QWPoAG5jEe1g1NEgxNTkMExxJE3ImKFbjgBW1FwAQnGJyfyFcOXEcIfH+SDnvBmcyjkQ8K92CcQGkcFscO1UHwSuxeqxxDAhugGDBaMgABzWRzwmhwIZxVjteFwcHzF6OACOf3gu2cwQ61AAtFA4IxGL0cHAqehObtagckEcge8toV0FlciLvnBfv8XoK3o4YNCkSi0UhLEq4uh2Bl8BBCgKQCSAKy9HJbAAq0OCQuB8zyAEkoMJYDwwEQXABBF08GARTiArYmExAA)

```sh
--parser babel
```

**Input:**

```jsx
bifornCringerMoshedPerplexSawderornCringerMoshedPerplexSawderQwer(() => {}, 200)
```

**Output:**

```jsx
bifornCringerMoshedPerplexSawderornCringerMoshedPerplexSawderQwer(() => {},
200);

```
vs
**Prettier pr-10941**
[Playground link](https://deploy-preview-10941--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAjAlgMwgJygYR3SgHM4cBZCAZwAs4ATABXIAcAbOADwGUBDAO4NyuAkVLkqdRixwdu-IeQCKA8gAp1ASgAEAXgB8O4AF8ANDoBMABmtaQZkBFYx00aslB8cOCAKbeCB4ofOwCfACeHo6oOHxgANZwMDys8cQkyDA4AK5wjnAAtqiMwgwAMnykOXxkAGK4hXwwrqTIIHw5MBAOILQwhewA6rTo8NRpYHA8QWPoAG5jEe1g1NEgxNTkMExxJE3ImKFbjgBW1FwAQnGJyfyFcOXEcIfH+SDnvBmcyjkQ8K92CcQGkcFscO1UHwSuxeqxxDAhugGDBaMgABzWRzwmhwIZxVjteFwcHzF6OACOf3gu2cwQ61AAtFA4IxGL0cHAqehObtagckEcge8toV0FlciLvnBfv8XoK3o4YNCkSi0UhLEq4uh2Bl8BBCgKQCSAKy9HJbAAq0OCQuB8zyAEkoMJYDwwEQXABBF08GARTiArYmExAA)

```sh
--parser babel
```

**Input:**

```jsx
bifornCringerMoshedPerplexSawderornCringerMoshedPerplexSawderQwer(() => {}, 200)
```

**Output:**

```jsx
bifornCringerMoshedPerplexSawderornCringerMoshedPerplexSawderQwer(
  () => {},
  200
);

```

Comment (sosukesuzuki) [reply src/language-js/print/call-arguments.js:125]: I didn't notice this sorry. I got it. Thanks.

**Comment by fisker** [comment]
Maybe we can add a test?

**Comment by thorn0** [comment]
Or rather open an issue. The current output in https://github.com/prettier/prettier/pull/10941#discussion_r637873498 is a bug (but this PR is not a proper fix for it).

**Pull request closed without merging**